.changeset/chilled-bugs-travel.md

```diff
@@ -0,0 +1,5 @@
+---
+'@modern-js/main-doc': patch
+---
+
+fix dev server crashed when prepare api handler failed
```

Comment:
Chinese changeset is required.

Reply:
It would be nice to have the Chinese changeset.

But for external contributors, we may not need to require the Chinese changeset, as it is not friendly to non-Chinese native speakers.

Reply:
Should we disable the Chinese changeset check on  CI?

Reply:
yes, the Chinese changeset check should be removed, or only work for internal contributors.